model/command_response.go

```diff
@@ -37,6 +41,40 @@ func (o *CommandResponse) ToJson() string {
 	return string(b)
 }
 
+func (o *CommandResponse) IsValid() error {
```

Comment:
Can you please also confirm that `icon_url` is a valid URL?

Reply:
Yep!